pi/setup_pi.sh

```diff
@@ -135,5 +135,10 @@ else
   systemctl enable darkstat
   systemctl enable dhcp-helper
 
+  # restart Pi daily to reduce the number of manual restarts we need to do
+  echo "
+  0 20 * * * /sbin/shutdown -r now
```

Comment:
Oddly enough, the RPis are in British Summer Time (BST) instead of UTC, like I'd expected. So I think that using 20:00 will get us to noon most of the year (except when our daylight savings times are not aligned. But even then, it should be OK if this happens then).